packages/fast-usdc/src/update-settler-reference.core.js

```diff
@@ -0,0 +1,67 @@
+/** @file core-eval to update Advancer's reference to Settler */
+
+import { makeTracer } from '@agoric/internal';
+import { E } from '@endo/far';
+
+const trace = makeTracer('FUSD-3', true);
+
+/**
+ * @import {CopyRecord} from '@endo/pass-style';
+ * @import {ManifestBundleRef} from '@agoric/deploy-script-support/src/externalTypes.js';
+ * @import {BundleID} from '@agoric/swingset-vat';
+ * @import {BootstrapManifest} from '@agoric/vats/src/core/lib-boot.js';
+ * @import {FastUSDCCorePowers} from './start-fast-usdc.core.js';
+ */
+
+const { keys } = Object;
+
+/**
+ * @typedef {object} UpdateOpts
+ * @property {{bundleID: BundleID}} [fastUsdcCode]
+ */
+
+/**
+ * @param {BootstrapPowers & FastUSDCCorePowers} powers
+ * @param {object} [config]
+ * @param {UpdateOpts} [config.options]
+ */
+export const updateSettlerReference = async (
+  { consume: { fastUsdcKit } },
+  { options = {} } = {},
+) => {
+  trace('options', options);
+  const { fastUsdcCode = assert.fail('missing bundleID') } = options;
+  const kitPre = await fastUsdcKit;
```

Comment:
Question: there's no need to produce a new kit, since we're not changing private args as in `update-noble-ica`: https://github.com/Agoric/agoric-sdk/blob/48d14ff79d3aca03113770c13c24f18f701f28c7/packages/fast-usdc/src/update-noble-ica.core.js?

Reply:
right. the facets of the kit shouldn't change because they're durable and the `privateArgs` aren't changing here

please don't include "pre" `fastUsdcKit` makes sense but it's taken and long. so maybe just `fuKit`  ;-)